src/Http/Http/src/DefaultHttpContext.cs

```diff
@@ -2,17 +2,21 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.ComponentModel;
+using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Security.Claims;
 using Microsoft.AspNetCore.Http.Features;
 using Microsoft.AspNetCore.Http.Features.Authentication;
+using Microsoft.AspNetCore.Shared;
+using Microsoft.AspNetCore.WebUtilities;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.AspNetCore.Http;
 
 /// <summary>
 /// Represents an implementation of the HTTP Context class.
 /// </summary>
+[DebuggerDisplay("{DebuggerToString(),nq}")]
 public sealed class DefaultHttpContext : HttpContext
```

Comment:
It's because I needed to add new implementations on DefaultHttpContext and DefaultHttpResponse to get the reason phrase. Abstractions doesn't reference the WebUtilities assembly.

Reply:
I added a comment to make it clearer.